xrpl4j-client/pom.xml

```
@@ -69,6 +69,23 @@
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
```

Let's add the `test` scope here, just to be safe.

```suggestion
      <artifactId>assertj-core</artifactId>
      <scope>test</scope>
```